docs/blog/unettest.md

```diff
@@ -0,0 +1,288 @@
+# UNETTEST
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When I write software, there are many ways to ask a question. I could go to
+Stack Overflow or a smarter coworker, but I usually like to ask my computer
+first.  What do I do if I feel like my computer is making me wait [seven and a
+half million years](https://hitchhikers.fandom.com/wiki/Deep_Thought) to get an
+answer? 
+
+I would want to revisit the question of how to ask questions. If I want to know
+the Ultimate Answer to Life, the Universe, and Everything, I must ask a damn
+Ultimate Question. Otherwise I will end up with an answer like 42 that took tons
+of effort to calculate.
+
+42 is an interesting data point and it might be useful, but in order to figure
+out what's going on, I might also like to know the answer to Death, the
+Universe, and Everything. And I might like to ask a question about just the
+Universe on its own, maybe a question or two on that Everything bit, and perhaps
+a couple of questions about a bowl of petunias. I cannot afford to wait so long
+between each answer. I have things to do.
+
+I do not have time to build the Earth in order to ask my questions so instead I
```

Comment:
I'm going to be honest here and say I don't think you need the Hitchhiker's analogy. Even though it's fun, it distracts from the main point of the post. I think what is more important is making sure that your audience knows what TDD is and what NGINX is.